### ISSUE TITLE
rpn_bbox_loss 0.4+ and others all less than 0.05 

### ISSUE THREAD

**Issue by rcx986635** [matterport/Mask_RCNN#2565]
When I train the network, rpn_bbox_loss is too high, up to 0.4+. and others all ok less than 0.05 , my env is tf2.4.1 + python3.6, 
Anyone has the idea?

**Comment by kimile599** [comment]
have you tried playing with the anchor size and ratio? I ran into the same issue in previous cases. Some can be solved and the rpn_bbox loss can be further lowered. But others won't go lower.

**Comment by kimile599** [comment]
![rpnbbox](https://user-images.githubusercontent.com/53457043/119045890-bb43ad00-b957-11eb-9f49-32ac062f3522.png)
Here is the same problem i am dealing with

**Comment by SebastianNagy** [comment]
> ![rpnbbox](https://user-images.githubusercontent.com/53457043/119045890-bb43ad00-b957-11eb-9f49-32ac062f3522.png)
> Here is the same problem i am dealing with

Have you been able to solve this problem? I'm dealing with this problem too but the loss is higher in my case. I could really use the advice. Thanks

**Comment by nbeili** [comment]
Hello ,
I am having the same problem. I had change the Anchors size  and also learning rate but rpn_bbox_loss stays high.
does anyone have an insight how we can solve this issue ? or some hints?
I would be gratful, thanks.
